### PR TITLE
MARP-259 mattermost connector - update document

### PR DESCRIPTION
**Pull request by phhung-axonivy**

### CODE REVIEW

**Comment by phhung-axonivy** [comment]
Hi @ivy-sgi 
I create this pull request for the document.
Could you help me review it?

**Comment by ivy-sgi** [comment]
Hi @phhung-axonivy ! I like the connector very much - let's ask @andreasbalsiger what he thinks about the documentation.

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
-# MY-PRODUCT-NAME
+# mattermost-connector
 
-[![CI Build](https://github.com/axonivy-market/REPO-NAME/actions/workflows/ci.yml/badge.svg)](https://github.com/axonivy-market/REPO-NAME/actions/workflows/ci.yml)
+[![CI Build](https://github.com/axonivy-market/mattermost-connector/actions/workflows/ci.yml/badge.svg)](https://github.com/axonivy-market/mattermost-connector/actions/workflows/ci.yml)
 
-"YOUR SHORT DESCRIPTION GOES HERE"
+Mattermost is a messaging platform for teams, providing channels for organized communication and collaboration in workplaces. It's known for its ease of use and integration with other tools, helping teams streamline their workflows.
```
Comment (andreasbalsiger) [inline README.md:5]: image
This "short" description is too long. 

What about this: "Mattermost enhances team messaging for streamlined workflows and better collaboration."

File: mattermost-connector-product/README.md [hunk 2]
```diff
@@ -1,32 +1,39 @@
-<!--
-Dear developer!     
+# Mattermost Connector
 
-When you create your very valuable documentation, please be aware that this Readme.md is not only published on github. This documentation is also processed automatically and published on our website. For this to work, the two headings "Demo" and "Setup" must not be changed
--->
+Mattermost is a messaging platform for teams, providing channels for organized communication and collaboration in workplaces. It's known for its ease of use and integration with other tools, helping teams streamline their workflows.
```
Comment (andreasbalsiger) [inline mattermost-connector-product/README.md:3]: Please rework this part. Try to stick on the general wording having a intro saying what the connector does (for Axon Ivy, and not just in general) and then have some bullet-points at the end.

File: mattermost-connector-product/README.md [hunk 3]
```diff
@@ -1,32 +1,39 @@
...
 
-# Mattermost Connector
+## Demo
 
-YOUR DESCRIPTION GOES HERE: Please just give a short description here without further headings.
+1. Call Slash command on the masttermost instance's chat
```
Comment (andreasbalsiger) [inline mattermost-connector-product/README.md:7]: Is "call" the right wording? isn't it something like "hit the slash command key on the mastermost instance's chat"?

**Review by andreasbalsiger — COMMENTED** [review]
see my comments.

**Comment by phhung-axonivy** [comment]
Hi @andreasbalsiger 
I have updated the document by your comments.
If it's good now, please approve the pull request. I can process to next step.

**Pull request merged**